internal/shim-sgx/src/handler.rs

```diff
@@ -367,8 +369,19 @@ impl<'a> SyscallHandler for Handler<'a> {
 
     // Stub for get_attestation() pseudo syscall
     // See: https://github.com/enarx/enarx-keepldr/issues/31
-    fn get_attestation(&mut self) -> sallyport::Result {
+    fn get_attestation(
+        &mut self,
+        nonce: *const libc::c_void,
+        nonce_len: libc::size_t,
+        buf: *const libc::c_void,
+        buf_len: libc::size_t,
+    ) -> sallyport::Result {
         self.trace("get_att", 0);
-        Ok([0.into(), SGX_TECH.into()])
+
+        let req = request!(SYS_GETATT => nonce, nonce_len, buf, buf_len);
```

Comment:
you are still missing the Block Cursor action before and after this

Reply:
Ah okay, I figured since it is the stub I would do it in the next PR, but I might as well do it here.

Reply:
@haraldh I updated it to use the Block. There may be a cleaner way to do it, but it should work... it copies the `nonce` to the block first, then allocates space for the `buf` so that it can get a pointer to the area right after the nonce. Afterwards it didn't seem like I can use the same pointers, so I made a new block, allocated the `nonce` to skip that area and get a pointer to the `buf` to copy data out of the block.

Reply:
Actually, I am thinking maybe I should test it before this is merged.